flink-batch-connectors/flink-hbase/src/main/java/org/apache/flink/addons/hbase/TableInputFormat.java

```diff
@@ -237,7 +244,7 @@ private void logSplitInfo(String action, TableInputSplit split) {
 	 *        End key of the region
 	 * @return true, if this region needs to be included as part of the input (default).
 	 */
-	private static boolean includeRegionInSplit(final byte[] startKey, final byte[] endKey) {
+	protected boolean includeRegionInSplit(final byte[] startKey, final byte[] endKey) {
```

Comment:
This function is according to the documentation intended so people can override it in a subclass. You cannot overrule a static function (and especially not if it is private).
http://stackoverflow.com/questions/2223386/why-doesnt-java-allow-overriding-of-static-methods